numpy/core/tests/test_umath.py

```diff
@@ -2126,6 +2126,12 @@ def test_nextafterf():
 def test_nextafterl():
     return _test_nextafter(np.longdouble)
 
+def test_nextafter_0():
+    for t, direction in itertools.product(np.sctypes['float'], (1, -1)):
+        tiny = np.finfo(t).tiny
+        assert_(0. < direction * np.nextafter(t(0), t(direction)) < tiny)
```

Comment:
Yes, for all floats we support, because they all support denormal numbers.  `tiny` is the smallest not-denormal number the type supports.